Cargo.lock

```diff
@@ -33,15 +33,48 @@ dependencies = [
 
```

Comment:
These changes are automatically done after running cargo update. Yes, I and Ionut had a chat on this. Looks like we need to update that to 1.56. But for this to happen we need to make sure that latest version of OpenSUSE Leap uses 1.56 or above. I have already asked Guillaume Gardet. Waiting for his response.

Reply:
Yeah, depending on what exactly changed in those two dependencies that we bumped, it could end in tens/hundreds of other dependencies getting bumped/added/removed... Such is life in Cargo Land.